packages/pwa-kit-cli/src/configs/webpack/config.js

```diff
@@ -101,8 +101,6 @@ const baseConfig = (target) => {
                 optimization: {
                     minimize: mode === production
                 },
-                // Always use source map, makes debugging the server much easier.
-                devtool: 'source-map',
```

Comment:
Removing this configuration option effects all the build configurations, not only the client and server. This means build for client optional, request processor, and renderer no longer have `source-map`. 

For some of those it might make sense not to have source maps, like the renderer, but I think request processor needs source maps for development, and the client optional needs source maps for development as well.